Automattic/jetpack
# Social: Fix Instagram image validation

There is an issue with Instagram's image validation, the name of the service is `instagram-business` and not `instagram`.

## Proposed changes:

* Fixed naming issue

### Other information:


## Jetpack product discussion


1204719803223144-as-1204804341013321
## Does this pull request change what data or activity we track or use?


no.
## Testing instructions:






* Add an instagram connection, and check that the validation is working as exptected in the block editor
 * It should get disabled if there is no image
 * It should get disabled if the image is larger than 4 Mb
 * It should get disabled if the image is not jpg
